xio-core/src/main/java/com/xjeffrose/xio/http/ProxyRouteConfig.java

```diff
@@ -64,4 +65,101 @@ public ProxyRouteConfig(Config config) {
     endsWith(config, "proxyPath", "/");
     // validatePath(config.getString("proxyPath"), config.origin());
   }
+
+  public ProxyRouteConfig(
+      List<HttpMethod> methods,
+      String host,
+      String path,
+      String permissionNeeded,
+      List<ClientConfig> clientConfigs,
+      ProxyHostPolicy proxyHostPolicy,
+      String proxyHost,
+      String proxyPath) {
+    super(methods, host, path, permissionNeeded);
+    this.clientConfigs = clientConfigs;
+    this.proxyHostPolicy = proxyHostPolicy;
+    this.proxyHost = proxyHost;
+    this.proxyPath = proxyPath;
+  }
+
+  public static Builder newBuilder(ProxyRouteConfig fallbackObject) {
+    return new Builder(fallbackObject);
+  }
+
+  /**
+   * Used to create a ProxyRouteConfig at runtime.
+   *
+   * <p>If a value is not set, it defaults to using the default object's value.
+   */
+  public static class Builder {
+    private ProxyRouteConfig fallbackObject;
```

Comment:
Same note as before either default or fallback as the name.

Reply:
fixed